ui/src/app/views/workspace/component.html

```diff
@@ -1,18 +1,18 @@
 <div *ngIf="!notFound else notFoundMessage">
   <div *ngIf="!workspaceLoading; else workspacePageLoading" style="display: flex;">
     <h2>{{workspace.name}}</h2>
-    <clr-button-group class="permissions-group btn-primary">
+    <clr-button-group class="permissions-group">
       <clr-button [disabled]="!writePermission" (click)="edit()"
           class="btn btn-icon">
-        <clr-icon class="is-solid" shape="pencil"></clr-icon>
+        <img src="assets/icons/edit@2x.png" alt="Edit" class="icon">
       </clr-button>
       <clr-button (click)="share()" class="btn btn-icon">
-        <clr-icon class="is-solid" shape="users"></clr-icon>
+        <img src="assets/icons/share@2x.png" alt="Share" class="icon">
```

Comment:
We probably want `svg`, not `png`. Failing that, we'll need different sizes and resolutions for the images - on my screen, for example, these already are starting to look grainy.  It _may_ help to set the `height` and `width` attributes - the html `img` element attributes, not the css properties - but eventually, assuming we don't use svg - and maybe if we do - I think we would want to investigate `sizes` and `srcset`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img